src/main/io.cpp

```diff
@@ -144,15 +145,75 @@ bool Dissolve::loadInputFromString(std::string_view inputString)
 // Load input from supplied file
 bool Dissolve::loadInput(std::string_view filename)
 {
+    std::ofstream output("C:/ProjectDissolve/dissolve/build/Release/output.toml");
+
     // Open file and check that we're OK to proceed reading from it
     LineParser parser(&worldPool());
     if (!parser.openInput(filename))
         return false;
 
     auto result = loadInput(parser);
-
     if (result)
     {
+        toml::basic_value<toml::discard_comments, std::map, std::vector> root;
+        toml::basic_value<toml::discard_comments, std::map, std::vector> masterNode;
+        if (coreData_.nMasterBonds() > 0)
```

Comment:
```suggestion
        if (!coreData_.nMasterBonds().empty())
```

This occurs a couple of times in the function

Reply:
I might be doing something wrong but this suggestion does not seem to work.